NewForce-at-Mountwest/bangazon-workforce-bucatini
# Db create department

Description
Created the CreateDepartment method within the DepartmentRepository. This allows the ability to create a new department

To create a department, type into the URL after locallhost 'department/create' and you will be presented with an empty form which includes entries for the name of the department and the budget. Once the form is filled out click on the respective affordance and the new department will be created

Related Ticket(s)
6. HR should be able to create a department

Steps to Test
git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout db-create-department to access my branch.
If you don't see the most recent version of their files, try git pull origin db-create-department
Run your server on my branch and make sure everything looks okay.

Check verbiage and comments to make sure they are legible and understandable.

Checklist:
[x ] I have commented my code, particularly in hard-to-understand areas
[x ] I have made corresponding changes to the documentation
[x ] My changes generate no new errors
